tests/Controller/CRUDControllerTest.php

```diff
@@ -1665,10 +1668,11 @@ public function testEditActionAjaxError(): void
 
     /**
      * @legacy
-     * @expectedDeprecation In next major version response will return 406 NOT ACCEPTABLE without `Accept: application/json`
      */
     public function testEditActionAjaxErrorWithoutAcceptApplicationJson(): void
     {
+        $this->expectDeprecation('In next major version response will return 406 NOT ACCEPTABLE without `Accept: application/json` or `Accept: */*`');
```

Comment:
Please, move this call directly before the line where the deprecation is expected to be triggered.